server/entrypoint.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
```

Comment:
Looks like its not related to this PR, please revert

Reply:
Sorry! Still a bit clumsy with git. I think I've fixed it here.

Reply:
I wonder would it be better to create a new PR?

Reply:
looks good now